aconcli/cmd/export.go

```diff
@@ -15,12 +15,13 @@ import (
 var ignoreSig bool
 
 var releaseCmd = &cobra.Command{
-	Use:   "export MANIFEST",
-	Short: "Export the specified ACON images into a tarball file",
+	Use:   "export <manifest file>",
+	Short: "Export the ACON image into a tarball file",
 	Long: `
-Export the specified ACON workloads into a tarball file, including the
-workload manifests, the certificates, the signatures and their file
-system layers`,
+Export the ACON image corresponding to the specified manifest file into
+a tarball. The image includes a manifest file, a signature file, a
+certificate file to verify the signature, and the file system layers
+of the image`,
```

Comment:
Please clarify if `export` (guess it doesn't) exports images on which the specified image/manifest depends.

A bit off topic here, what's the recommended way to figure out and export dependencies (i.e., images that a specified image depends on)?

Reply:
Resolved. 

If in the manifest file, there are alias names which cannot be resolved locally (i.e. in current manifest file), then we have dependencies. Currently, the user needs to track which imported images are used as the dependencies.